Applications/DisplayProperties/main.cpp

```diff
@@ -12,12 +16,28 @@ int main(int argc, char** argv)
 
     auto window = GWindow::construct();
     window->set_title("Display Properties");
-    window->move_to(100,100);
+    window->move_to(100, 100);
     window->resize(400, 448);
     window->set_resizable(false);
     window->set_main_widget(instance.root_widget());
     window->set_icon(load_png("/res/icons/16x16/app-display-properties.png"));
 
+    // Let's create the menubar first
+    auto menubar = make<GMenuBar>();
+
+    auto app_menu = GMenu::construct("Display Properties");
+    app_menu->add_action(GAction::create("Exit", { Mod_Alt, Key_F4 }, [&](const GAction&) {
```

Comment:
Please use `GCommonActions` for this:
```suggestion
    app_menu->add_action(GCommonActions::make_quit_action([&](const GAction&) {
```

Reply:
Oh that's super cool. I'll get on updating all of the files that use the original version (I think `Terminal` still does)

Reply:
I don't think there are any :)

Reply:
> I don't think there are any :)

Must be an issue with my fork, you know that me and git don't mix that well ;)